x-pack/platform/plugins/shared/observability_ai_assistant/server/routes/knowledge_base/route.ts

```diff
@@ -294,31 +294,22 @@ const importKnowledgeBaseEntries = createObservabilityAIAssistantServerRoute({
       throw new Error('Knowledge base is not ready');
     }
 
-    const limiter = pLimit(5);
-    const promises = resources.params.body.entries.map(async (entry) => {
-      return limiter(async () => {
-        return pRetry(
-          () => {
-            return client.addKnowledgeBaseEntry({
-              entry: {
-                confidence: 'high',
-                is_correction: false,
-                public: true,
-                labels: {},
-                role: KnowledgeBaseEntryRole.UserEntry,
-                ...entry,
-              },
-            });
-          },
-          { retries: 10 }
-        );
-      });
-    });
+    const entries = resources.params.body.entries.map(entry => ({
+      confidence: ConversationConfidenceEnum.high,
```

Comment:
Let's use the constant instead of the enum, so we don't have to depend on `@kbn/elastic-assistant-common`. We want to get rid of both `confidence` and `is_correction` sooner rather than later anyway